message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java

```diff
@@ -4164,6 +4170,13 @@ public void setDate(Time date)
 
 		} // setDate
 		
+
+		@Override
+		public void setInstant(Instant instant) {
+			Time time = m_timeService.newTime(instant.toEpochMilli());
+			setDate(time);
+		}
+		
```

Comment:
You could just do it one line. Small thing, I know, but still ...

Reply:
OK